test/test10.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+export LD_LIBRARY_PATH=build/
+if build/marc --debug output=test.bin; then
```

Comment:
Are you testing lack of required argument? In that case the other argument should be --output. Else this fails but not as per the test.